Automattic/jetpack
# Boost: track when someone moves from Boost Cache to WPSC

Track sites that move from using Boost to perform caching to WPSC.

Fixes #

## Proposed changes:

* Add a hook on the action "jb_cache_moved_to_wpsc" that performs a tracks event.

### Other information:


## Jetpack product discussion
pc9hqz-2QH-p2
pc9hqz-2V8-p2

## Does this pull request change what data or activity we track or use?


Yes, this PR allows us to track customers who go from using the Cache module of Boost to WP Super Cache.

## Testing instructions:
* Edit Page_Cache.php and add an error_log in track_move_to_wpsc() so you can see that functions being called.
* Disable WP Super Cache.
* Enable Boost, and the Cache module.
* Enable WP Super Cache and visit the settings page. Note the warning message about Boost.
* Disable the Boost Cache module. In the future this will delete advanced-cache.php
* Delete wp-content/advanced-cache.php. Do not disable Boost.
* Refresh the WPSC settings page.
* Note the error log showing the hooked function track_move_to_wpsc fired.
